addons/sale_management/views/sale_portal_templates.xml

```diff
@@ -64,9 +64,9 @@
                                             </div>
                                         </strong>
                                     </td>
-                                    <td class="text-center" t-if="sale_order._can_be_edited_on_portal() and report_type == 'html'">
+                                    <td class="text-end" t-if="sale_order._can_be_edited_on_portal() and report_type == 'html'">
                                         <a t-att-data-option-id="option.id" href="#" class="mb8 js_add_optional_products d-print-none" aria-label="Add to cart" title="Add to cart">
-                                            <span class="fa fa-shopping-cart"/>
+                                            <span class="btn btn-secondary">Add to order</span>
```

Comment:
![image](https://github.com/odoo/odoo/assets/77771734/8bb3c3e5-ca3c-446f-bd18-5d2b2be2ca1e)
With adding secondary line it increase the width of line and because of that it does not display very good
What we can do is decrease height button or align item middle vertically :thinking: 
@dalaOdoo @dit-odoo could I have your opinion here?

Reply:
I prefer what we have now

CC @dit-odoo